api/v1/helpers/verbs/doFind.js

```diff
@@ -71,10 +71,15 @@ function doFindResponse(reqResNext, props, opts, resultObj) {
   .then((retval) => {
     u.sortArrayObjectsByField(props, retval);
 
+    // Merge props and opt fields to exclude
+    let fieldsToExclude = opts.fieldsToExclude ? opts.fieldsToExclude : [];
+    fieldsToExclude = props.fieldsToExclude ? fieldsToExclude.fieldsToExclude
```

Comment:
I think lines 75-76 would be easier to read like this:

```
const fieldsToExclude = [].concat(opts.fieldsToExclude || [],
  props.fieldsToExclude || [])
```

Reply:
Wow, awesome. Thanks! Really like the 'defensive' way in JS using "|| default_value". Will try next time 🙂